lib/kennel/models/monitor.rb

```diff
@@ -108,6 +108,16 @@ def as_json
         @as_json = data
       end
 
+      # resolve composite monitors ... only works when referenced monitors already exist
+      # since leaving names or bad ids in the query breaks the monitor update
+      def resolve_linked_tracking_ids(id_map)
+        if as_json[:type] == "composite"
```

Comment:
can we be certain that :type will always exist? Should we handle that case here?

Reply:
if it's nil then it's not `== "compisite"` 🤷‍♂

Reply:
true... right - I as using fetch() before:)